index.js

```diff
@@ -1505,6 +1505,29 @@
   }
   S.pipe = def('pipe', {f: [Z.Foldable]}, [f(Fn($.Any, $.Any)), a, b], pipe);
 
+  //# pipeK :: (Foldable f, Chain m) => f (Any -> m Any) -> m a -> m b
+  //.
+  //. Takes an array of functions assumed to be unary which return
+  //. values with a [Chain][], and a value of that Chain, and returns the
+  //. result of applying the sequence of transformations to the initial value.
```

Comment:
We could tweak the line wrapping to make the lengths of these lines more balanced:

```javascript
  //. Takes an array of functions assumed to be unary which return values
  //. with a [Chain][], and a value of that Chain, and returns the result
  //. of applying the sequence of transformations to the initial value.
```

Reply:
Fixed